### DISCUSSION TITLE
How to cancel a Boost?

### DISCUSSION THREAD

**Discussion by barryperfect** [tomasmcguinness/homeassistant-mixergy#64] [Q&A]
I notice that you can cancel a boost from the Mixergy App, but there isn't a Action to do this in the integration.

Am I right in saying the way to cancel a boost is to set the required charge level to the current or lower than the current charge level?

The reason I want to cancel a boost is during a Octopus Free Electricity session I want to set the charge to 100% and if that has not been reached by the end of the session, to cancel the boost.

**Comment by tomasmcguinness** [comment]
@barryperfect Sorry Barry - I'm not really keeping on top of these discussions. You're right - you can just drop the target charge level down below the current. 

I'll look to add a cancel_boost service. It makes sense.

**Reply by barryperfect** [reply]
Thanks Tom, now I know setting a lower charge cancels the boost I am OK with it as it is, but it would be handy to have a command to do the same if you are updating the integration for another reason :-)
